src/mumble/AudioWizard.h

```diff
@@ -17,6 +18,8 @@ class AudioWizard : public QWizard, public Ui::AudioWizard {
 private:
 	Q_OBJECT
 	Q_DISABLE_COPY(AudioWizard)
+
+	bool hasUsableEchoCancellation(AudioInputRegistrar *air, QString outputSys);
```

Comment:
Missing Docs

Reply:
I think the function name is pretty well self-documented. But I can add an additional description anyway.

Reply:
The important bit would probably be to shortly explain what "usable" means in this context. Without knowing that there are multiple echo cancellation option this might not be self-explanatory :point_up: